spec.md

```diff
@@ -1032,27 +1043,32 @@ if it contains sensitive information.
 ## Updating a Service Instance
 
 By implementing this endpoint, Service Broker authors can enable users to
-modify two attributes of an existing Service Instance: the Service Plan and
-parameters. By changing the Service Plan, users can upgrade or downgrade their
-Service Instance to other Service Plans. By modifying parameters, users can change
-configuration options that are specific to a Service Offering or Service Plan.
+modify the following attributes of an existing Service Instance:
+* The Service Plan (allowing users to upgrade of downgrade their Service
```

Comment:
typo here, should be `upgrade **or** downgrade`

Reply:
Thank you @AartiKriplani !